src/app/components/Loop/QuoteModal.tsx

```diff
@@ -0,0 +1,264 @@
+  getLoopOutQuote: typeof getLoopOutQuote;
+  getLoopOut: typeof getLoopOut;
+  getLoopIn: typeof getLoopIn;
+}
+
+interface OwnProps {
+  amt: string;
+  sct: string;
+  isOpen?: boolean;
+  type: string;
+  dest: string;
+  srf: string;
+  sf: string;
+  mf: string;
+  pre: string;
+  chan: string;
+  adv: boolean;
+  htlc: boolean;
+  onClose(): void;
+}
+
+type Props = StateProps & DispatchProps & OwnProps;
+
+class QuoteModal extends React.Component<Props> {
+  componentWillUpdate(nextProps: Props) {
+    if (!this.props.isOpen && nextProps.isOpen) {
+      // Fire even if amt is in store in case we need to cycle
+      this.props.getLoopOutQuote(this.props.amt, this.props.sct);
+    }
+  }
+  render() {
+    const {
+      loopQuote,
+      amt,
+      sct,
+      adv,
+      isOpen,
+      onClose,
+      type,
+      hasPassword,
+      error,
+    } = this.props;
+    if (loopQuote === null) {
+      return null;
+    }
+    const actions: ButtonProps[] = [
+      {
+        children: (
+          <>
+            <Icon type="lightning" theme="filled" /> {`${type}`}
+          </>
+        ),
+        type: 'primary' as any,
+      },
+    ];
+    if (loopQuote === null) {
+      return null;
+    }
+    const isVisible = !!isOpen && !!(hasPassword || error);
+
+    let content;
+    if (loopQuote.miner_fee !== '') {
+      content = (
+        <>
+          <div className="QuoteModal">
+            <p>{`Miner fee: ${loopQuote.miner_fee} sats`}</p>
+            <p>{`Prepay amt: ${
+              loopQuote.prepay_amt === undefined ? '1337' : loopQuote.prepay_amt
+            } sats`}</p>
+            <p>{`Swap fee: ${loopQuote.swap_fee} sats`}</p>
+            <p>{`Swap amt: ${amt} sats`}</p>
+            <p>{`Sweep Conf. Target: ${sct}`}</p>
+
+            {/* Default Loop  */}
+            {adv === false &&
+              actions.map((props, idx) => (
+                <Button
+                  key={idx}
+                  {...props}
+                  onClick={this.props.type === 'Loop Out' ? this.loopOut : this.loopIn}
+                />
+              ))}
+            {/* Advanced Loop */}
+            {adv === true &&
+              actions.map((props, idx) => (
+                <Button
+                  key={idx}
+                  {...props}
+                  onClick={
+                    this.props.type === 'Loop Out' ? this.advLoopOut : this.advLoopIn
+                  }
+                />
+              ))}
+          </div>
+        </>
+      );
+    } else if (error) {
+      content = (
+        <Alert
+          type="error"
+          message="Failed to get quote"
+          description={error.message}
+          showIcon
+        />
+      );
+    }
+    return (
+      <Modal
+        title={`${type} Quote`}
+        visible={isVisible}
+        onCancel={onClose}
+        okButtonProps={{ style: { display: 'none' } }}
+        centered
+      >
+        <div className="QuoteModal">{content}</div>
```

Comment:
There's a div above with `QuoteModal` as a classname too, you probably don't want both of those.

Reply:
changed classname